index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> ULTRA-PALLOZA</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/6a7b1e2152.js" crossorigin="anonymous"></script>
+</head>
+<body>
+<!-- HOME PAGE -->
+<!-- HAMBURGER-WINDOW start-->
+
+<div  class="hambu-wind not-active"> 
+  <div class="x-btn"> 
+    <button type="btn-close" id="close">
+      <img src="/bg-img/blackclose.png" alt="x">
+    </button>
+  </div>
+  <a class="pop-window" href="/about.html">About</a>
+  <a class="pop-window" href="#about">Program</a>
+  <a class="pop-window" href="#footer"> Join</a>   
+  <a class="pop-window" href="#footer"> Sponsors</a>  
+  <a class="pop-window" href="#footer"> News</a>  
+</div>
+
+<!-- HAMBURGER-WINDOW end-->
+
+  <!-- SECTION HERO -->
+<section id="section-hero">
+  <div id="hero-div">
+    <div class="social-media">
+      <i class="fa-brands fa-square-facebook fa-2x"></i>
+      <i class="fa-brands fa-square-twitter fa-2x"></i>
+    </div>
+    <nav id="navbar">
+      <div id="nav-content"> 
+        <a id="ham-link" >
+        <img id="ham-btn" src="/bg-img/Hamburger_icon.svg.png" alt="hamburger-icon">
+        </a>
+        <a id="logo-mobile"> <img id="logo-btn-show" src="/bg-img/logo.png" alt="logo-icon"> </a>
+        <a id="logo-link" href="/index.html">
+          <img id="logo-btn" src="/bg-img/logo.png" alt="logo-icon">
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
```

Comment:
- [ ] I personally loved your design, you did a great job 💯 But according to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Colors, Typography, and Layout. So kindly, change the `background-color` in the top nav to `black`, and add the rest of links (English, My page, Logout). I also suggest to remove the `font-weight` property from the nav-items to match the original design 👍

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/197103787-dff4fdee-98e1-4dc0-ad1b-51ca746e00a1.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/197103753-1c63d24b-2ace-46db-883e-4849cbb0d0fd.PNG" width="500" /> |